cmd/clusterctl/client/cluster/mover.go

```diff
@@ -741,7 +741,7 @@ func pauseClusterClass(ctx context.Context, proxy Proxy, n *node, pause bool, mu
 
 	patchHelper, err := patch.NewHelper(clusterClass, cFrom)
 	if err != nil {
-		return errors.Wrapf(err, "error creating patcher for ClusterClass %s/%s", n.identity.Namespace, n.identity.Name)
```

Comment:
This seems useful information to keep?

Reply:
Nevermind, I see this is covered in the inner calls

Reply:
Yup. I kept additional wrapping only if it provided additional information